site/src/components/card/Card.tsx

```diff
@@ -28,26 +29,28 @@ const Card = ({
   url,
   linkText,
   keylineColor,
-  theme = "dark",
+  disableKeylineAnimation = false,
```

Comment:
Might be easier to read `keyLineAnimation = 'true'` so you don't have to do `!disableKeylineAnimation` on line 52 (instead just `keylineAnimation`), but not a blocker

Reply:
Makes sense, I'll amend this on my upcoming PR